examples/02-plot/ortho-slices.py

```diff
@@ -19,12 +19,12 @@
 mesh = examples.download_embryo()
 mesh.bounds
 
-##############################################################################
+# %%
```

Comment:
Do these similar changes in 5 files lead to the same rendered example? Probably yes, because as of this branch tip no such comments are left in `examples/`, just making sure.

Reply:
Ah, separate commit in https://github.com/pyvista/pyvista/pull/6375/commits/222cdb61dbf103ca4584c0a631ccf7f7078ac58c explains it, thanks.